tests/ZfcRbacTest/Guard/RoutePermissionsGuardTest.php

```diff
@@ -410,7 +410,7 @@ public function testProperlyFillEventOnAuthorization()
 
     public function testProperlySetUnauthorizedAndTriggerEventOnUnauthorization()
     {
-        $eventManager = $this->getMock('Zend\EventManager\EventManagerInterface');
+        $eventManager = $this->getMock('Zend\EventManager\EventManager');
```

Comment:
Two questions:
1) Why is this needed?
2) How is this a BC break, when it's only in the tests?